ted/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -259,7 +265,21 @@ private void initializeCameraSubsystem(){
       System.out.println("FAIL: initializeCamera");
     }
   }
-  
+
+  /**
+   * A method to init the LEDSubsystem
+   */
+  private void initializeLEDSubsystem(){
+    if(InstalledHardware.LEDSInstalled){
+      subsystems.setLEDSubsystem(new LEDSubsystem(Constants.ledCanID, LEDStripType.RGB));
```

Comment:
The LEDStripType.RGB should be a constant that should be in Constants.java. So, Constants.LEDStripType should be set to LEDStripType.RGB and then Constants.LEDStripType should be used here.

Reply:
done